<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLoadConnectome</class>
 <widget class="QDialog" name="DialogLoadConnectome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Connectome Matrix</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>CMAT file</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditCMAT">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonOpenCMAT">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="freeview.qrc">
         <normaloff>:/resource/icons/file_open_16.png</normaloff>:/resource/icons/file_open_16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Parcellation file</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEditParcel"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonOpenParcel">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="freeview.qrc">
         <normaloff>:/resource/icons/file_open_16.png</normaloff>:/resource/icons/file_open_16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Color table file</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineEditCTAB"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonOpenCTAB">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="freeview.qrc">
         <normaloff>:/resource/icons/file_open_16.png</normaloff>:/resource/icons/file_open_16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="freeview.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogLoadConnectome</receiver>
   <slot>OnOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonOpenCMAT</sender>
   <signal>clicked()</signal>
   <receiver>DialogLoadConnectome</receiver>
   <slot>OnButtonOpenCMAT()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonOpenParcel</sender>
   <signal>clicked()</signal>
   <receiver>DialogLoadConnectome</receiver>
   <slot>OnButtonOpenParcel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonOpenCTAB</sender>
   <signal>clicked()</signal>
   <receiver>DialogLoadConnectome</receiver>
   <slot>OnButtonOpenCTAB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnOK()</slot>
  <slot>OnButtonOpenCMAT()</slot>
  <slot>OnButtonOpenParcel()</slot>
  <slot>OnButtonOpenCTAB()</slot>
 </slots>
</ui>
